### ISSUE TITLE
Rename "Server Profile" context menu entry to something else

### ISSUE THREAD

**Issue by nearcatch** [jaimeadf/BetterDiscordPlugins#212]
Discord has per-server profiles that let you edit your nick/avatar per server. The context menu for that is "Edit Server Profile". The entry for the GuildProfile plugin should be changed to something less similar. Maybe "Server Info"?

**Comment by Yilmaz4** [comment]
Yeah, the "Edit Server Profile" can be confused with "Server Profile" option. But in my opinion, the option name should stay as it is because the "server profile" is intended to look pretty similar to user profiles. Renaming it to "Server Info" (for example) may not make sense. _In my opinion._
